bot/services/pin_service.py

```diff
@@ -0,0 +1,104 @@
+from typing import Union
+
+import discord
+from discord import RawMessageDeleteEvent
+from discord.ext.commands import Context
+
+from bot.consts import Staff, Colors
+from bot.data.class_repository import ClassRepository
+from bot.data.pin_repository import PinRepository
+from bot.messaging.events import Events
+from bot.models.class_models import ClassPin
+from bot.services.base_service import BaseService
+from bot.sock_bot import SockBot
+from bot.utils.helpers import fetch_optional_message
+
+PIN_REACTION = '📌'
+MIN_PIN_REACTIONS = 5
+MAX_CONTENT_CHARS = 250
+MAX_PINS_PER_CHANNEL = 50
+
+
+class PinService(BaseService):
+
+    def __init__(self, *, bot: SockBot):
```

Comment:
What's the point of the * here

Reply:
Hm, good question. It was like this in the `example_service.py` so I just followed it. I'll remove it.